agent/xds/testdata/clusters/terminating-gateway-hostname-service-subsets.golden

```diff
@@ -65,18 +65,6 @@
         "endpoints": [
           {
             "lbEndpoints": [
-              {
```

Comment:
nit: I think all of the terminating gateway cluster tests are using hostnames now? And nothing is using EDS?

Reply:
The `web` service still has IP addresses: https://github.com/hashicorp/consul/blob/tgw-unique-hostnames/agent/xds/testdata/endpoints/terminating-gateway-service-subsets.golden